acceptance/tests/import_export/legacy_storeconfigs_import_export.rb

```diff
@@ -2,6 +2,9 @@
 
   confine :except, :platform => 'ubuntu-10.04-amd64'
 
+  master_pup_ver = on(master, "facter puppetversion").stdout.chomp
```

Comment:
We might already prepopulate this ourselves, try facts(master.name)['puppetversion'] I think. We do it after Puppet installs, it saves on the continual calls to facter.